rfc-013-code-repositories.md

```diff
@@ -0,0 +1,77 @@
+# Code repository standards
+
+## Summary
+
+A set of standards to be used for projects that will be shared within Made Tech or the greater technology community. These are projects hosted on GitHub, Bitbucket, Gitlab or similar public source control management.
+
+## Problem
+
+Made Tech code repositories (to be referred to as repos herein) do not have a common look and feel.
+
+## Visibility
+
+The visibility of your project **SHOULD** mean the following:
+
+- You **MUST** set your repo to **Private to creator**  if code isn’t ready for wide consumption.
+- You **MUST** set your repo to **Private to the organisation** when the code is propriety or NDA bound.
```

Comment:
I think open by default can be good here too (maybe for parts of) a "proprietary system"